flexbox/horizontal_flexbox.go

```diff
@@ -2,7 +2,7 @@ package flexbox
 
 import "github.com/charmbracelet/lipgloss"
 
-// HorizontalFlexBox responsive box grid insipred by CSS flexbox
+// HorizontalFlexBox responsive box grid inspipred by CSS flexbox
```

Comment:
```suggestion
// HorizontalFlexBox responsive box grid inspired by CSS flexbox
```

Reply:
Thx! Very sharp 🧐 typos-ls didn't even catch it!